sirepo/package_data/static/js/sirepo-command.js

```diff
@@ -242,6 +251,11 @@ SIREPO.app.directive('commandTable', function(appState, commandService, latticeS
                 return -1;
             }
 
+            $scope.copyCommand = function(cmd) {
```

Comment:
As above, use arrow notation and omit parens:
```javascript
scope.copyCommand = cmd => {...}
```

Reply:
77045d74288ceb536b10500920e635863ad3d399